src/options.ts

```diff
@@ -124,7 +115,7 @@ function applyHttpOptionsDefaults(options: N9NodeRouting.Options): void {
 		typeof options.http.logLevel !== 'undefined'
 			? options.http.logLevel
 			: (tokens: morgan.TokenIndexer, req: express.Request, res: express.Response): string => {
-					const formatLogInJSON: boolean = (global as any).n9NodeRoutingData.formatLogInJSON;
+					const formatLogInJSON: boolean = options.enableLogFormatJSON;
```

Comment:
```suggestion
					const formatLogInJSON: boolean = options.log.formatJSON;
```
ça me semble plus cohérent de se baser sur ce qui a été décidé par l'init du logger

Reply:
Done